logs/README.txt

```diff
@@ -1,7 +0,0 @@
-Log files:
-
-apps_info.log = INFO level logging for all edx-ora2 apps and OpenAssessmentBlock
```

Comment:
Should this information be captured someplace else? It seems like it could be useful (assuming it is still true).